src/com/ableton/Pyenv.groovy

```diff
@@ -120,8 +120,8 @@ class Pyenv implements Serializable {
   protected void assertPyenvRoot() {
     assert pyenvRoot
 
-    if (!script.fileExists(pyenvRoot)) {
-      script.error "pyenv root path '${pyenvRoot}' does not exist"
+    if (script.sh(returnStatus: true, script: "${pyenvRoot}/bin/pyenv --version")) {
```

Comment:
So `script.dir` is fine with POSIX paths on Windows, but not `script.fileExists`... That's strange 🤔

Reply:
No, neither one of them works. The problem is with *absolute, CygWin-style paths*. I just ran this code in the sandbox:

```groovy
node('generic-win') {
    dir('C:\\Windows\\Temp') {
        bat 'dir'
    }
    if (fileExists('C:\\Windows\\Temp\\vmware-vmusr-ci.log')) {
        echo 'it is there'
    } else {
        echo 'it is not there'
    }
    
    dir('/c/windows/temp') {
        bat 'dir'
    }
    if (fileExists('/c/windows/temp/vmware-vmusr-ci.log')) {
        echo 'it is also there'
    } else {
        echo 'it is also not there'
    }
}
```

Here's the result:

```
Running on [generic-win-vs2019-00](https://newci.office.ableton.com/computer/generic%2Dwin%2Dvs2019%2D00/) in F:\workspace\Utilities\sandbox
 [Pipeline] {
 [Pipeline] dir
 Running in C:\Windows\Temp
 [Pipeline] {
 [Pipeline] bat
 
 C:\Windows\Temp>dir
  Volume in drive C has no label.
  Volume Serial Number is 4CEB-A7DB
 
  Directory of C:\Windows\Temp
 
 22/09/2023  13:29    <DIR>          .
 22/09/2023  13:29    <DIR>          ..
 17/09/2023  07:46         3.788.840 f03139a7-0df9-4c0e-8fae-a95623bd1935.tmp
 27/07/2023  13:23                 0 FXSAPIDebugLogFile.txt
 27/07/2023  13:23                 0 FXSTIFFDebugLogFile.txt
 22/09/2023  13:31           931.062 MpCmdRun.log
 22/09/2023  12:08           957.232 MpSigStub.log
 22/09/2023  09:46           517.550 msedge_installer.log
 20/09/2023  09:51    <DIR>          vmware-SYSTEM
 14/09/2023  23:19           345.309 vmware-vmsvc-SYSTEM.log
 26/07/2023  11:29            44.173 vmware-vmusr-ci.log
 14/09/2023  21:33    <DIR>          WinSAT
 14/09/2023  14:34    <DIR>          {5ED6833A-C365-4743-9B64-12A039EC1E06}
                8 File(s)      6.584.166 bytes
                5 Dir(s)  15.565.709.312 bytes free
 [Pipeline] }
 [Pipeline] // dir
 [Pipeline] fileExists
 [Pipeline] echo
 it is there
 [Pipeline] dir
 Running in /c/windows/temp
 [Pipeline] {
 [Pipeline] bat
 
 F:\c\windows\temp>dir
  Volume in drive F has no label.
  Volume Serial Number is 0C9E-D458
 
  Directory of F:\c\windows\temp
 
 22/09/2023  14:49    <DIR>          .
 22/09/2023  14:49    <DIR>          ..
                0 File(s)              0 bytes
                2 Dir(s)  20.222.361.600 bytes free
 [Pipeline] }
 [Pipeline] // dir
 [Pipeline] fileExists
 [Pipeline] echo
 it is also not there
```

Reply:
> neither one of them works.

Then this means that https://github.com/Ableton/python-pipeline-utils/pull/94/commits/59e794c124a1aff0c263c95c9255d5ef97509bc2 breaks running `VirtualEnv.cleanup` on Windows.

Reply:
Good catch. Fixed in 37fc6a4.